src/profession.cpp

```diff
@@ -221,7 +221,7 @@ void profession::check_item_definitions( const itypedecvec &items ) const
 {
     for( auto &itd : items ) {
         if( !item::type_is_defined( itd.type_id ) ) {
-            debugmsg( "profession %s: item %s does not exist", id.c_str() , itd.type_id.c_str() );
+            debugmsg( "profession %s: item %s does not exist", id.c_str(), itd.type_id.c_str() );
```

Comment:
Strange, the origin of the extra space is the ancient 8888c70. What version of astyle does the build bot use? Maybe it was upgraded recently?

Also, there seems to be an astyle regression in profession.h (see the build log of #20206) (so why did this build not fail?!).